Godeps/_workspace/src/github.com/contiv/contivmodel/network.json

```diff
@@ -17,6 +17,13 @@
 					"title": "Tenant Name",
 					"length": 64
 				},
+				"nwType": {
+					"type": "string",
+					"format": "^(infra|data)$",
+					"default": "\"data\"",
```

Comment:
Didn't notice this while reviewing the contivmodel change doesn't `"default": "data",` work? do you need the `\"`?